src/plugins/condition/components/Condition.vue

```diff
@@ -246,6 +246,7 @@ export default {
             this.domainObject.configuration.criteria.push(criteriaObject);
         },
         dragStart(e) {
+            e.dataTransfer.setData('dragging', e.target); // required for FF to initiate drag
```

Comment:
Does this need to be unset on dragStop or osmething? Just curious...

Reply:
Good call. Added a dragStop method to clear data

Reply:
Don't know what happened to it. Added it back.